### ISSUE TITLE
Feature Request: allow multiple source directories to be specified on command line

### ISSUE THREAD

**Issue by jclark-dot-org** [cesarParra/apexdocs#35]
We are using sfdx unlocked packages to install some of our utility classes, and we recently started using the [unpackaged metadata][1] feature to move all of our test classes and test-only metadata out of the installed package.  This results in a structure like this:

```
/package-root
  + /force-app
    + /main
      + /default
        + /classes
          | ApexClass.cls
  + /unpackaged-test-metadata 
    + /classes
      | ApexClassTest.cls
```

I'd like to run `apexdocs` once for both `force-app/main/default/classes/` and `unpackaged-test-metadata/classes`, since I'm using `@see` in the class to reference the test class, and vice versa.  I tried using the top-level project directory with recursion:

```
npm exec -- apexdocs-generate --s .  
```

But that missed `unpackaged-test-metadata` and picked up things in `node_modules`.  Then I tried specifying two directories:

```
npm exec -- apexdocs-generate -s force-app/ -s unpackaged-test-metadata
```

But that resulted in an (unsurprising) error: 

```
node:internal/fs/utils:670
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received an instance of Array
    at Object.readdirSync (node:fs:1385:10)
    at getClassesFromDirectory (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/Command/Generate.js:37:34)
    at Object.generate (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/Command/Generate.js:29:21)
    at Object.<anonymous> (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/cli/generate.js:52:41)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'ERR_INVALID_ARG_TYPE'
}

Node.js v17.2.0
```

**Note**:  You could argue that we don't need to document test classes, and I might agree - I'm still trying to decide best practices for apexdocs.  However, the same issues will apply for any project which includes multiple packages.  See for example  [this sample sfdx-project.json][2] in the `sfdx` docs, which contains 4 packages with dependencies.

[1]: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_unpackaged_md.htm
[2]: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_extract_dependency_info.htm

**Comment by jongpie** [comment]
As another person that uses ApexDocs, I'd also really love to have this feature. I'm currently using ApexDocs for my project [Nebula Logger](https://github.com/jongpie/NebulaLogger/), which currently has 3 separate packages (1 "core" package, with 2 "plugin" packages). I'm using ApexDocs to generate the [Apex documentation](https://jongpie.github.io/NebulaLogger/apex/) for the project, but it currently only includes the "core" package - I'd love to be able to specify multiple directories so that I can include the plugins' docs in the resulting markdown.

**Comment by cesarParra** [comment]
I like this, and the use case makes a lot of sense. I'll see what I can do!

**Comment by cesarParra** [comment]
I just published v1.14.0 with this functionality.

Multiple directories can be specified using the same `--sourceDir` (or `-s` alias) separated by spaces, for example

`apexdocs-generate -s src1 src2 -t docs`
